integration/airflow/tests/integration/requests/failed_sql_extraction.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "eventType": "START",
```

Comment:
I think we should always check for both START and COMPLETE events.

Reply:
Done.